aspnetcore/security/authentication/accconfirm.md

```diff
@@ -1,38 +1,66 @@
 ---
-title: Account Confirmation and Password Recovery
+title: Account Confirmation and Password Recovery in ASP.NET Core
 author: rick-anderson
 description: Shows how to build an ASP.NET Core app with email confirmation and password reset.
 keywords: ASP.NET Core, password reset, email confirmation, security
 ms.author: riande
 manager: wpickett
-ms.date: 06/14/2017
+ms.date: 07/19/2017
 ms.topic: article
-ms.assetid: d794500b-86f7-4229-a237-e0dd00e2dc08
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: security/authentication/accconfirm
 ---
-# Account confirmation and password recovery
-
-<a name=security-authentication-account-confirmation></a>
+# Account confirmation and password recovery in ASP.NET Core
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 This tutorial shows you how to build an ASP.NET Core app with email confirmation and password reset.
 
 ## Create a New ASP.NET Core Project
 
-The tutorial requires Visual Studio 2017 or higher.
+# [ASP.NET Core 2.x](#tab/aspnet20)
+
+The tutorial requires Visual Studio 2017 preview 2 or later.
```

Comment:
Capitalize the "p" on "preview". Also, should this sentence explicitly state that these steps apply to Windows only?